WebRoot/js/zimbraMail/abook/controller/ZmContactController.js

```diff
@@ -283,6 +283,7 @@ function() {
 ZmContactController.prototype._setViewContents =
 function() {
 	var cv = this._contactView;
+	this.editingContactList = this._contact.list;
```

Comment:
We need to extract current i.e editing contact list while setting edit view. Because contact list can get changed from cache  while adding contacts to contact group.